src/main/java/de/tum/in/www1/artemis/web/rest/AssessmentResource.java

```diff
@@ -170,12 +170,15 @@ public AssessmentResource(AuthorizationCheckService authCheckService, UserReposi
         Exercise exercise = exerciseRepository.findByIdElseThrow(exerciseId);
         final var exampleSubmission = exampleSubmissionRepository.findBySubmissionIdWithResultsElseThrow(submissionId);
 
+        var user = userRepository.getUserWithGroupsAndAuthorities();
+        var isInstructor = authCheckService.isAtLeastInstructorForExercise(exercise, user);
+        var isTutor = authCheckService.isAtLeastTeachingAssistantForExercise(exercise, user);
         // It is allowed to get the example assessment, if the user is an instructor or
         // if the user is a tutor and the submission is not used for tutorial in the assessment dashboard
-        boolean isAllowed = authCheckService.isAtLeastInstructorForExercise(exercise)
-                || authCheckService.isAtLeastTeachingAssistantForExercise(exercise) && !Boolean.TRUE.equals(exampleSubmission.isUsedForTutorial());
-        if (!isAllowed) {
-            forbidden();
+        // The reason is that example submissions with isTutorial = false should be shown immediately (with the assessment) to the tutor and
+        // for example submission with isTutorial = true, the assessment should not be shown to the tutor. Instead, the tutor should try to assess it him/herself
+        if (!(isInstructor || isTutor && !exampleSubmission.isUsedForTutorial())) {
```

Comment:
```suggestion
        if (!(isInstructor || (isTutor && !exampleSubmission.isUsedForTutorial()))) {
```
This makes it easier to understand to everyone who is not familiar with the order of evaluation